<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkInfoDemoClass</class>
 <widget class="QDialog" name="NetworkInfoDemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NetworkInfoDemo</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btnGetHostInfo1">
         <property name="text">
          <string>获取本机名称的IP地址</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnGetHostInfo2">
         <property name="text">
          <string>获取本机名称的IP地址(异步)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnGetHostInfo3">
         <property name="text">
          <string>获得www.baidu.com域名的IP地址(异步)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbOnlyIPv4">
         <property name="text">
          <string>只显示IPv4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="lstAddress"/>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="btnGetNetworkInterface">
         <property name="text">
          <string>获得主机的所有网络接口(适配器)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="lstInterface"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="NetworkInfoDemo.qrc"/>
 </resources>
 <connections/>
</ui>
